src/components/RatesDisplay.js

```diff
@@ -1,33 +1,56 @@
 
 import React, { Component } from 'react';
-import { Text, View } from 'react-native';
-
+import { Text, View, Image, ActivityIndicator } from 'react-native';
 import { RateScreenStyles as styles } from '../styles';
-
+import { Query } from 'react-apollo';
+import { sortTodayRates } from '../dataApi';
+import { GET_TODAY_RATES } from '../operations';
+import { RatesDisplayItem } from '.';
+import Images from '@assets/images';
+import Whoops from './Whoops';
+import moment from 'moment';
 export default class Home extends Component {
```

Comment:
Could you please change the name of this component?

Reply:
I can.